Dockerfile

```diff
@@ -27,7 +27,7 @@ ENV DOCKER_API_VERSION=1.32
 
 # Install ansible, boto, aws-cli, and some handy tools
 RUN echo "===> Installing Utilities from apk ..."  && \
-    apk -v --update --progress add sudo git bash wget openssh groff less python py-pip curl jq unzip nodejs nodejs-npm coreutils python py-pip openssl ca-certificates make sshpass openssh-client rsync gnupg gettext && \
+    apk -v --update --progress add sudo git bash wget openssh groff less python py-pip curl jq unzip nodejs nodejs-npm coreutils python py-pip openssl ca-certificates make sshpass openssh-client rsync gnupg gettext openjdk8-jre=8.191.12-r0 && \
```

Comment:
isn't jdk 8 too old? why not install later version?

Reply:
There's no official support for jdk 11 on alpine. There is some port for that. 
There is now support for 12 and 13 but both are in "early access" stage